__tests__/app.test.js

```diff
@@ -82,4 +82,32 @@ describe("GET", () => {
         });
     });
   });
+  describe('/api/articles', () => {
+    test('STATUS 200, should return an array of article objects', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const {articles} = response.body;
+          expect(typeof articles).toBe('object')
+          expect(articles[0]['created_at']).toBe('2020-11-03T09:12:00.000Z')
```

Comment:
So you're only checking the first article here, what kind of method could you use to check every object has the correct keys?

Reply:
should have used map. Will fix it.

Reply:
forEach actually